scim2-sdk-common/src/main/java/com/unboundid/scim2/common/utils/Parser.java

```diff
@@ -161,6 +177,76 @@ public static Filter parseFilter(final String filterString)
     return readFilter(new StringReader(filterString.trim()), false);
   }
 
+  /**
+   * Add parser options, in order to modify parser behavior within the current
+   * thread.
+   *
+   * <p>NOTE: SCIM server implementations are not guaranteed to support a given option.</p>
+   * <p>NOTE: these should be removed when no longer needed.</p>
+   *
+   * <pre>
+   *   try
+   *   {
+   *     Parser.addOptions(ParserOption.ALLOW_SEMICOLONS_IN_ATTRIBUTE_NAMES();
```

Comment:
There's a typo at the end of this that makes ALLOW_SEMICOLONS_IN_ATTRIBUTE_NAMES look like a method call.